R/zzz.R

```diff
@@ -1,10 +1,13 @@
-.onLoad <- function(libname, pkgname){
-
-  assign("stage", "production", envir = parent.env(environment()))
-  assign("base_url",
-         "https://equity-tool-api.urban.org",
-         envir = parent.env(environment()))
+.onAttach <- function(libname, pkgname) {
+  stage <- "production"
```

Comment:
Thanks for sharing the best practice for using .onAttach for startup messages - we agree that the approach you've implemented comports with the best practice